### DISCUSSION TITLE
Any idea how to block Opera from adding autorun entry when upgrading?

### DISCUSSION THREAD

**Discussion by Destroy666x** [chocolatey-community/chocolatey-packages#2279] [Q&A]
It's doing that after each upgrade and then if I forget to remove the entry with Autoruns, my restarted PC loads 50+ tabs of a browser I don't use regularly.

**Comment by pauby** [comment]
Do you mean the browser automatically starts when you login (i.e. automatically runs)?

**Reply by Destroy666x** [reply]
Yes

**Reply by pauby** [reply]
There will likely be some PowerShell code we could add to the package to prevent this.

However, I'd suggest if you want this functionality that you raise a PR to do add it.

**Reply by Destroy666x** [reply]
The thing is here I didn't even know if there's a switch, but found it in this updated post where it wasn't earlier: https://www.reddit.com/r/operabrowser/wiki/opera/installer_commands/ `--run-at-startup=0`. I'll submit a PR for a parameter for that once I'll have time.
